src/lib/Scenes/City/Components/SavedEventSection/index.tsx

```diff
@@ -4,19 +4,28 @@ import PinSavedOff from "lib/Icons/PinSavedOff"
 import PinSavedOn from "lib/Icons/PinSavedOn"
 import SwitchBoard from "lib/NativeModules/SwitchBoard"
 import React, { Component } from "react"
-import { Image, TouchableWithoutFeedback } from "react-native"
+import { Image, TouchableOpacity, TouchableWithoutFeedback } from "react-native"
 import styled from "styled-components/native"
 
 export interface Props {
   data: any
   citySlug: string
+  sponsoredContentUrl: string
 }
 
 export class SavedEventSection extends Component<any> {
   handleTap = () => {
     SwitchBoard.presentNavigationViewController(this, `/city-save/${this.props.citySlug}`)
   }
 
+  navigateToBMWArtGuide() {
+    const { sponsoredContentUrl } = this.props
+    SwitchBoard.presentNavigationViewController(
+      this,
+      sponsoredContentUrl || "https://www.bmw-arts-design.com/bmw_art_guide"
```

Comment:
Since the `SavedEventSection` _always_ gets its data, we can drop the `|| "https:..."` here because `sponsoredContentUrl` will never be `nil`.

I like checking for `nil`, though! Good to guard against error cases. Can't wait to enable some more of those TypeScript flags... /cc @alloy